plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaTopicDescription.java

```diff
@@ -80,6 +81,30 @@ public String getTopicName()
         return message;
     }
 
+    @Override
```

Comment:
Please add a unit test for this. To make sure everything top to bottom implements equals.